catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -437,6 +443,10 @@ public void deleteRegistryEntriesByMetacardIds(List<String> metacardIds,
                 .collect(Collectors.toList());
         List<Filter> filters = getBasicFilter();
         Filter filter = filterBuilder.allOf(filters);
+        if (includeInternal) {
+            filter = filterBuilder.anyOf(filter,
```

Comment:
❓ Do we ever expect to have more internal-representation metacards? Should we account for that now, perhaps adding a `Set` of internal tags to `RegistryConstants` and populating it today with `REGISTRY_TAG_INTERNAL`?